main/REPL/scratch.md

```diff
@@ -0,0 +1 @@
+**tyg todo: Need pointer to where home.scratch is defined and explained**
```

Comment:
Here is the definition: https://github.com/Agoric/agoric-sdk/blob/master/packages/cosmic-swingset/lib/ag-solo/vats/scratch.js

It's not explained in this file, but the summary is that you can use `get`, `set`, and `list` to save key-value pairs for later. It is only on the ag-solo and is not accessible from the chain, making it private to the user of the ag-solo. Deploy scripts are ephemeral, so it's important to be able to use `scratch` for saving objects in a deploy script for a later script to use.